pm_dftb_opt_write/pm_dftb_opt_write.xml

```diff
@@ -0,0 +1,116 @@
+<tool id="pm_dftb_opt_write" name="Generate dftb+ outputs (dftb+)" version="0.1.0" python_template_version="3.5">
+    <requirements>
+        <requirement type="package">dftbplus</requirement>
+        <requirement type="package">pymuonsuite</requirement>
+        <requirement type="package">zip</requirement>
+    </requirements>
+    <command detect_errors="exit_code"><![CDATA[
+        pip show pymuonsuite || ( git clone https://github.com/muon-spectroscopy-computational-project/pymuon-suite /tmp/pymuon-suite && pip install /tmp/pymuon-suite && rm -rf /tmp/pymuon-suite) &&
+        unzip $folder &&
+        if test -f "params.yaml"; then echo "params.yaml present"; else echo "params.yaml missing" && exit 64; fi 
+        && if test -f "input.cell"; then echo "input.cell present"; else echo "input.cell missing" && exit 64; fi &&
+        out_folder="`python ${__tool_directory__}/get_out_folder.py`" &&
+        bash ${__tool_directory__}/run.sh \$out_folder ${__tool_directory__} &&
+        if grep -q 'pbc-0-3' params.yaml; then cp ${__tool_directory__}/README-pbc \$out_folder/README; elif grep -q '3ob-3-1' params.yaml; then cp ${__tool_directory__}/README-3ob \$out_folder README; fi; 
```

Comment:
This should catch the case where the param set isn't defined in the YAML and make sure the default set's README gets used

Reply:
added, adds both files with no input